renku/service/cache/__init__.py

```diff
@@ -27,3 +28,19 @@ class ServiceCache(
     """Service cache manager."""
 
     pass
+
+
+def make_cache():
+    """Create cache structure."""
+    sub_dirs = [CACHE_UPLOADS_PATH, CACHE_PROJECTS_PATH]
+
+    for subdir in sub_dirs:
+        if not subdir.exists():
+            subdir.mkdir()
```

Comment:
I'd rather write `subdir.mkdir(parents=True, exist_ok=True)` and remove `if not subdir.exists():`.

Reply:
Done. Ty!